plugins/linux-v4l2/data/locale/en-GB.ini

```diff
@@ -13,4 +13,5 @@ ColorRange.Default="Default"
 ColorRange.Partial="Partial"
 ColorRange.Full="Full"
 CameraCtrls="Camera Controls"
-
+AutoresetOnTimeout="Reset On Timeout"
+FramesUntilTimeout="Frames Until Timeout"
```

Comment:
Do not include changes to locale files other than en-US.  All translation work is done via Crowdin, as noted in [our Contributing Guidelines](https://github.com/obsproject/obs-studio/blob/4a890e1da9a27003c796527b4a603312514e9eac/CONTRIBUTING.rst#quick-links-for-contributing).